src/nvim/ex_docmd.c

```diff
@@ -138,9 +136,7 @@ struct dbg_stuff {
 # define ex_language            ex_ni
 #endif
 
-/*
- * Declare cmdnames[].
- */
+/// Declare cmdnames[].
```

Comment:
I don't think a doxygen comment does anything here.
```suggestion
// Declare cmdnames[].
```